src/EFCore.PG/ValueGeneration/UUid7ValueGenerator.cs

```diff
@@ -0,0 +1,59 @@
+using System.Runtime.InteropServices;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.ValueGeneration;
+
+/// <summary>
+///     Generates sequential <see cref="Guid" /> values according to the UUID version 7 specification.
+///     Will be updated to use Guid.CreateVersion7 when available.
+/// </summary>
+public class UUid7ValueGenerator : ValueGenerator<Guid>
```

Comment:
**Suggestion:** The .NET team has chosen to stick to avoid using multiple synonyms, hence `Guid.CreateVersion7`. For consistency, I would go with `GuidVersion7ValueGenerator`.

Reply:
**Question:** All generators in the current package are prefixed with `Npgsql`. Should we follow that convention or not, @roji?

Reply:
Moreover, they are marked as internal APIs. Should we rather be consistent with them or with EF Core's own `GuidValueGenerator`?

Reply:
Let's have the prefix, for consistency with similar existing types in the package, and to reinforce the pubternal-ness some more.